nft/nfts.yaml

```diff
@@ -1453,14 +1453,6 @@ paths:
               default: eth-mainnet
       parameters:
         - $ref: '#/components/schemas/apiKey'
-      servers:
```

Comment:
do we not want to keep servers objects at the method level?

Reply:
this one was duplicate